templates/docs/examples/patterns/tiered-list/50-50-desktop-with-description-cta.html

```diff
@@ -0,0 +1,88 @@
+{% extends "_layouts/examples.html" %}
+{% from "_macros/tiered-list.jinja" import tiered_list %}
+
+{% block title %}Tiered list / 50/50 on desktop with description CTA{% endblock %}
+
+{% block standalone_css %}patterns_all{% endblock %}
+
+{% block content %}
+{%- call(slot) tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
+  {%- if slot == 'title' -%}
+    <h2>H2 - up to two lines; ideally one.</h2>
+  {%- endif -%}
+
+  {%- if slot == 'description' -%}
+    <p>A <a href="#">private cloud</a> provides organisations with on-demand
+    compute, storage and other resources that can be accessed over the network
+    and that are reserved exclusively for them - either in their own data centre
+    or via a 3rd party.</p>
+
+    <div class="p-cta-block">
```

Comment:
It may help to put an HTML comment above these optional CTA blocks so that it's clearer to the reader which exact section of markup is being added to create the CTA block.
See this example from Quote wrapper PR:
<img width="778" alt="Screenshot 2024-08-21 at 3 26 18 PM" src="https://github.com/user-attachments/assets/0d92e6e0-2457-4eb7-a987-3eeed573be02">